src/test/scala/cognite/spark/v1/SequencesRelationTest.scala

```diff
@@ -332,44 +335,47 @@ class SequencesRelationTest
     cleanupSequences(Seq(id))
 
     exception.getMessage should
-      include("Column valueType cannot be modified: the previous value is STRING and the user attempted to update it with LONG")
+      include(
+        "Column valueType cannot be modified: the previous value is STRING and the user attempted to update it with LONG")
   }
 
   it should "chunk sequence if more than 10000 columns in the request" in {
-    val key = shortRandomString()
-
-    //650 * 200 give us 130000 columns in total
-    val ids = (1 to 650).map(_ => UUID.randomUUID().toString)
-    try {
-      val sequencesToInsert = ids.map(
-        id =>
-          SequenceInsertSchema(
-            externalId = Some(id),
-            name = Some(s"a|${key}"),
-            columns = (1 to 200).map(
-              i =>
-                SequenceColumnCreate(
-                  name = Some("col" + i),
-                  externalId = "col" + i,
-                  description = Some("col1 description"),
-                  valueType = "STRING",
-                  metadata = Some(Map("foo" -> "bar"))
-              ))
-        ))
-
-      ingests(sequencesToInsert)
-
-      val sequences = retryWhile[Array[Row]](
-        spark.sql(s"select * from sequences where name = 'a|$key'").collect,
-        _.length != 650)
-      sequences.length shouldBe 650
-      sequences.head.getAs[String]("name").startsWith("a|") shouldBe true
+    Seq("abort", "upsert").foreach { mode =>
+      val key = shortRandomString()
 
-    } finally {
+      //650 * 200 give us 130000 columns in total
+      val ids = (1 to 650).map(_ => UUID.randomUUID().toString)
       try {
```

Comment:
why did we have a try/catch here? cannot follow easily.

Reply:
Just to make sure we delete the sequence in case the test failed, kind of cleaning up thing :)